python/ray/util/state/state_cli.py

```diff
@@ -398,6 +399,11 @@ def ray_get(
         :class:`RayStateApiException <ray.util.state.exception.RayStateApiException>`
             if the CLI is failed to query the data.
     """  # noqa: E501
+    if not id:
+        raise click.UsageError(
```

Comment:
is `BadParameter` a better error?

Reply:
Update https://github.com/ray-project/ray/pull/43649/commits/90f9003b29d147927ff261cd6805a78b85e641a7. The error message is still the same.